lib/src/widgets/homework/homeworkList.dart

```diff
@@ -15,6 +15,7 @@ class HomeworkList extends StatelessWidget {
       padding: EdgeInsets.symmetric(horizontal: 5.0, vertical: 0.0),
       child: ListView.builder(
         shrinkWrap: true,
+        physics: NeverScrollableScrollPhysics(),
```

Comment:
What does this do?

Reply:
Makes it so that the list is not scrollable, mainly because it's in another `ListView` so you have a scrollable list inside a scrollable list, and that's a wonky interface